regionService/cloud-regionsrv.spec

```diff
@@ -17,20 +17,19 @@
 #
 
 Name:           cloud-regionsrv
-Version:        7.0.1
+Version:        8.0.0
 Release:        0
 License:        GPL-3.0+
 Summary:        Cloud Environment Region Service
 URL:            https://github.com/SUSE/Enceladus
 Group:          Productivity/Networking/Web/Servers
 Source0:        %{name}-%{version}.tar.bz2
 Requires:       apache2
-Requires:       apache2-mod_wsgi     >= 4.2.8
+Requires:       apache2-mod_wsgi-python3     >= 4.2.8
 Requires:       openssl
-Requires:       python
-Requires:       python-Flask
-Requires:       python-netaddr
```

Comment:
Don't need this dependency anymore?

Reply:
they are now python3- dependencies and netaddr has been replaced by ipaddress which is built in